pkg/lite-apiserver/proxy/handler.go

```diff
@@ -45,28 +51,64 @@ type EdgeServerHandler struct {
 	cacher *RequestCacheController
 }
 
-func NewEdgeServerHandler(config *config.LiteServerConfig, manager *cert.CertManager, cacher *RequestCacheController) (http.Handler, error) {
-	e := &EdgeServerHandler{
-		apiserverUrl:  config.KubeApiserverUrl,
-		apiserverPort: config.KubeApiserverPort,
-		manager:       manager,
-		storage:       storage.NewFileStorage(config),
-		cacher:        cacher,
-		timeout:       config.BackendTimeout,
+func NewEdgeServerHandler(config *config.LiteServerConfig, certManager *cert.CertManager, cacher *RequestCacheController) (http.Handler, error) {
+	h := &EdgeServerHandler{
+		apiserverUrl:    config.KubeApiserverUrl,
+		apiserverPort:   config.KubeApiserverPort,
+		certManager:     certManager,
+		reverseProxyMap: make(map[string]*EdgeReverseProxy),
+		storage:         storage.NewFileStorage(config),
+		cacher:          cacher,
+		timeout:         config.BackendTimeout,
+	}
+
+	h.initProxies()
+
+	return h.buildHandlerChain(h), nil
+}
+
+func (h *EdgeServerHandler) initProxies() {
+	h.defaultProxy = NewEdgeReverseProxy(h.certManager.DefaultTransport(), h.apiserverUrl, h.apiserverPort, h.timeout, h.storage, h.cacher)
+
+	for commonName, transport := range h.certManager.GetTransportMap() {
+		proxy := NewEdgeReverseProxy(transport, h.apiserverUrl, h.apiserverPort, h.timeout, h.storage, h.cacher)
```

Comment:
add some leveled logs here.

Reply:
fixd in #44 as that pr refactor transport and cert